src/Rules/VimeoVideoId.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Vimeo\Laravel\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Vimeo\Laravel\Facades\Vimeo;
+
+class VimeoVideoId implements Rule
+{
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $response = Vimeo::request('/videos/'.$value, [], 'GET');
```

Comment:
I would also check that the id is a valid unsigned bigint as well before the request

It also doesn't handle exceptions